nni/experiment/experiment.py

```diff
@@ -69,7 +69,7 @@ def __init__(self, tuner: Tuner, training_service: str) -> None:
             A tuner instance.
         training_service
             Name of training service.
-            Supported value: "local", "remote", "openpai".
+            Supported value: "local", "remote", "openpai", "aml", "kubeflow", "frameworkcontroller", "adl".
```

Comment:
so we support all the training services in v2.0?

Reply:
It seems we don't support `heterogeneous` and `DLTS`? @liuzhe-lz

Reply:
you can add "and hybrid training service."

Reply:
fix it